### PR TITLE
Updated client.go. Added K8sClient to clients struct

### PR DESCRIPTION
**Pull request by yprokule**
Methods for nodes cordoning, uncordoning and draining added

### CODE REVIEW

File: pkg/nodes/drain.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/kubectl/pkg/drain"
+)
+
+// NewDrainHelperBuilder builds drain Helper that contains parameters to control the behaviour of drain.
+func NewDrainHelperBuilder(
```
Comment (kononovn) [inline pkg/nodes/drain.go:16]: I think:
Drain/Cordon/Uncordon functions have to a part of node.Builder struct.
Please add these methods as part of Builder in node.go file

**Review by kononovn — CHANGES_REQUESTED** [review]
Please check comment above

**Review by kononovn — CHANGES_REQUESTED** [review]
PR looks good. It's almost ready to merge.
Can you please do following changes:

Lave 2 commits instead of 3 commits under this PR.
 
The first commit has to have following message:
"Bump dependence. Add kubectl drain lib"
Second commit message:
"Updated client.go. Added K8sClient to clients struct."

**Review by trewest — APPROVED** [review]
/lgtm

**Pull request merged**